controllers/tasks.js

```diff
@@ -52,8 +53,15 @@ const fetchTasks = async (req, res) => {
  */
 const getUserTasks = async (req, res) => {
   try {
+    const { status } = req.query
     const { username } = req.params
-    const allTasks = await tasks.fetchUserTasks(username)
+    let allTasks = []
+
+    if (!Object.values(TASK_STATUS).includes(status)) {
+      return res.boom.notFound('Status not found!')
+    }
+
+    allTasks = await tasks.fetchUserTasks(username, status ? [status] : [])
```

Comment:
What if I send `?status=active+completed` in query params? I suppose it should be [...status] instead of [status], what do you think? How will we handle multiple statuses?

Reply:
I thought of handling it like this ? https://github.com/Real-Dev-Squad/website-backend/pull/369#discussion_r650407769

Reply:
Okay. That's our assumption for now. Let's go ahead with that then 👍🏻

Reply:
Wait, if it's going to just one status at a time, why are we even wrapping it inside a list? It should simply be a string, right?

Reply:
> why are we even wrapping it inside a list? It should simply be a string, right?
didn't get you with this ?

> That's our assumption for now. Let's go ahead with that then 👍🏻

Didn't get should we go ahead with `?status=active+completed` or `?status=active,completed` or should not support multiple status. I have implemented earlier with this `?status=active,completed` but @sumitd94 said its invalid. So i reverted the code. Let me know with which approach we should move ahead.